### PR TITLE
redfish: Do not request the Advantech BMC to reboot

### PR DESCRIPTION
**Pull request by hughsie**
It reboots automatically.

Based on a patch by Vincent Tang <vincent.tang@advantech.com.tw>, many thanks.

Fixes https://github.com/fwupd/fwupd/issues/6536

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [X] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: plugins/redfish/README.md [hunk 1]
```diff
@@ -57,6 +57,13 @@ default 0ms.
 
 Since: 1.8.0
 
+### Flags=no-manager-reset-request
+
+The BMC device will auto-reboot and so fwupd should not explicitly call
+`/redfish/v1/Managers/1/Actions/Manager.Reset`.
+
+Since: 1.9.11
```
Comment (superm1) [inline plugins/redfish/README.md:65]: 2.0.0

Comment (superm1) [reply plugins/redfish/README.md:65]: (Unless of course planning to backport this to 1_9_X, then maybe that makes sense)

Comment (hughsie) [reply plugins/redfish/README.md:65]: Yup, was planning to.

Comment (hughsie) [reply plugins/redfish/README.md:65]: I think we should use the needs-backport tag for stuff like this that shouldn't wait the [couple of months?] before we do 2.0.0

Comment (superm1) [reply plugins/redfish/README.md:65]: > I think we should use the needs-backport tag for stuff like this that shouldn't wait the [couple of months?] before we do 2.0.0

sounds good

**Review by vincent-tang1 — APPROVED** [review]

**Pull request merged**